content/applications/websites/livechat/responses.rst

```diff
@@ -1,48 +1,176 @@
+   :align: center
+   :alt: View of the message generated from using the /help command in Odoo Live Chat.
+
+.. seealso::
+  - :doc:`/applications/productivity/discuss/overview/get_started`
+  - :doc:`/applications/productivity/discuss/overview/team_communication`
+
+Helpdesk & Helpdesk search
+--------------------------
+
+The `/helpdesk` and `/helpdesk_search` commands allow operators to both create helpdesk tickets
+directly from a conversation, and search through existing tickets by keyword or ticket number.
 
 .. important::
-   | - For *helpdesk tickets*: make sure the application is installed on your database and the
-     option *Live Chat* under :menuselection:`Helpdesk --> Configuration --> Helpdesk Teams`
-     is enabled.
-   | - For *leads*: the *CRM* application must be installed on your database.
+   The `/helpdesk` and `/helpdesk_search` commands can only be used if the *Helpdesk* app has been
+   installed, and *Live Chat* has been activated on a *Helpdesk* team. To activate :guilabel:`Live
+   Chat`, go to :menuselection:`Helpdesk application --> Configuration --> Teams`, and select a
+   team. Scroll to the :guilabel:`Channels` section and check the box labeled :guilabel:`Live Chat`.
 
-To access the ticket or lead created from the chat, click on the shortcut link.
+Create a ticket from a live chat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. image:: responses/create_ticket.png
+If an operator types `/helpdesk` in the chat window, the conversation is used to create a *Helpdesk*
+ticket.
+
+After entering the `/helpdesk` command, select a title for the ticket and type it into the chat
+window, then press `Enter`.
+
+.. image:: responses/helpdesk.png
    :align: center
-   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat
+   :alt: View of the results from a helpdesk search in a Live Chat conversation.
 
-.. tip::
-   Helpdesk tickets created from the chat automatically add the conversation as a description of
-   the ticket. The same goes for the creation of a lead.
+The newly created ticket will be added to the *Helpdesk* team that has live chat enabled. If more
+than one team has live chat enabled, the ticket will automatically be assigned based on the team's
+priority.
+
+The transcript from the conversation will be added to the new ticket, under the
+:guilabel:`Description` tab.
+
+To access the new ticket, click on the link in the chat window, or go to the
+:menuselection:`Helpdesk app` and click the :guilabel:`Tickets` button on the kanban card for the
+appropriate team.
+
+Search for a ticket from a live chat
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If an operator types `/helpdesk_search` in the chat window, they can search through *Helpdesk*
+tickets by ticket number or keyword.
+
+After entering the `/helpdesk_search` command, type a keyword or ticket number, then press `Enter`.
+If a related ticket is found, a link will be generated in the conversation window.
```

Comment:
What if the operator searches by keyword  and there are multiple entries found?
Are all of the entries seen by the customer, or only the operator?
If there are multiple entries, does the operator have to select one to share with the customer?